doc/Makefile

```
@@ -45,13 +48,18 @@ clean:
 	rm -rf generated/*
 	rm -rf modules/*
 
+sync_git:
+	git clone https://github.com/benchopt/benchmark_logreg_l2 $(BENCH_LOGREG_PATH) > /dev/null 2>&1 || (cd $(BENCH_LOGREG_PATH) ; git pull && cd ../../doc)
+	git clone https://github.com:benchopt/benchmark_lasso $(BENCH_LASSO_PATH) > /dev/null 2>&1 || (cd $(BENCH_LASSO_PATH) ; git pull && cd ../../doc)
+
 html-noplot:
 	$(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
 .PHONY: html
 html:
+	make sync_git
```

Just using dependencies:
```suggestion
html: sync_git
```